{"uuid":"eae51819-10a0-41d4-bbe7-5031a0520191","historyId":"6f64c116a8c23adfe0f82b785fa28794","fullName":"APi_Negative.Accounting.SG_NG_Analytics.Global_Temperature_Details","labels":[{"name":"package","value":"APi_Negative.Accounting.SG_NG_Analytics"},{"name":"testClass","value":"APi_Negative.Accounting.SG_NG_Analytics"},{"name":"testMethod","value":"Global_Temperature_Details"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"untitled"},{"name":"subSuite","value":"APi_Negative.Accounting.SG_NG_Analytics"},{"name":"host","value":"DESKTOP-L0TTVHB"},{"name":"thread","value":"16552@DESKTOP-L0TTVHB.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Global_Temperature_Details","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\tTest failed due to mismatch:\nExpected Status Code: 400, Actual Status Code: 200\nExpected Status Body: Fail, Actual Status Body: Fail","trace":"java.lang.AssertionError: The following asserts failed:\n\tTest failed due to mismatch:\nExpected Status Code: 400, Actual Status Code: 200\nExpected Status Body: Fail, Actual Status Body: Fail\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)\r\n\tat APi_Negative.Accounting.SG_NG_Analytics.Global_Temperature_Details(SG_NG_Analytics.java:352)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"arg3","value":"with negative"},{"name":"arg2","value":"Fail"},{"name":"arg1","value":"400"},{"name":"arg0","value":"-1"}],"start":1740047028632,"stop":1740047031690}